pkg/engine/lifecycletest/fuzzing/README.md

```diff
@@ -66,3 +66,55 @@ By default, reproduction test files will be written to a temporary directory. A
 specific directory can be configured using the
 `PULUMI_LIFECYCLE_TEST_FUZZING_REPRO_DIR` environment variable.
 :::
+
+### Running a suite of fuzz tests
+
+The `TestFuzz` test generates and tests a set of fixtures, and can be run with
+the top-level `test_lifecycle_fuzz` target:
+
+```
+make test_lifecycle_fuzz GO_TEST_RACE=false
+```
+
+`GO_TEST_RACE=false` is passed to disable race detection. You can omit this (and
+enable race detection) but the tests will run *significantly* more slowly and in
+general you probably won't want this.
+
+You can change how many checks Rapid performs by setting the
+`LIFECYCLE_TEST_FUZZ_CHECKS` Make variable on the command line. For instance, to
+perform 10,000 checks:
+
+```
+make test_lifecycle_fuzz GO_TEST_RACE=false LIFECYCLE_TEST_FUZZ_CHECKS=10000
+```
+
+If you use `go test` directly, make sure you set the
+`PULUMI_LIFECYCLE_TEST_FUZZ` variable to something truthy; the test will be
```

Comment:
```suggestion
`PULUMI_LIFECYCLE_TEST_FUZZ` variable to something; the test will be
```

Doesn't look like it needs to be truthy.

Reply:
Good shout.